introduction.md

```diff
@@ -6,33 +6,31 @@ layout: documentation
 
 # About openHAB
 
-openHAB is **software** for integrating different home automation systems and technologies into one single solution that allows over-arching automation rules, and that offers uniform user interfaces.
+openHAB is **software** for integrating different home automation systems, devices and technologies into one single solution.
+It provides uniform user interfaces, and a common approach to automation rules across the entire system, regardless of the number of manufacturers and sub-systems involved.
 
-But openHAB is also a **community** of users, contributors and maintainers.
+openHAB is also a **community** of users, contributors and maintainers, working together on an open-source, interoperable approach to home automation.
 
 ## openHAB Structure
 
-openHAB 2 is an open-source solution based on the [Eclipse SmartHome](https://eclipse.org/smarthome/) framework. It is written in Java and uses [Apache Karaf](http://karaf.apache.org/) together with [Eclipse Equinox](https://www.eclipse.org/equinox/) as an OSGi runtime and bundles this with [Jetty](https://www.eclipse.org/jetty/) as an HTTP server.
+openHAB 2 is based on the [Eclipse SmartHome](https://eclipse.org/smarthome/) framework, and is written in Java.
+It uses [Apache Karaf](http://karaf.apache.org/) together with [Eclipse Equinox](https://www.eclipse.org/equinox/) to create an Open Services Gateway initiative (OSGi) runtime environment.
+[Jetty](https://www.eclipse.org/jetty/) is used as an HTTP server.
 
-openHAB is highly modular software, which means that the base installation (the "runtime") can be extended through different kinds of "add-ons", either to communicate with new home automation solution, or to offer a new kind of user interface.
+openHAB is highly modular software, and can be extended through "add-ons".
+Add-ons give openHAB a wide array of capabilities, from User Interfaces, to the ability to interact with a large and growing number of physical Things.
 
-The structure of openHAB can be summarized this way:
+The overall architecture of openHAB is shown in the figure below:
 
 ![distribution overview](images/distro.png)
 
-Add-ons can come from three different origins:
-
-* Add-ons coming from the Eclipse SmartHome project, as Eclipse SmartHome is the base of openHAB 2,
-* Add-ons coming from the openHAB version 1 package, as openHAB 2 includes a compatibility layer,
-* Add-ons made for openHAB 2.
```

Comment:
Mayba add a oneliner that mentions the 3 addon sources?

Something like:

> As the image shows, you can get Add-ons from 3 different sources

This could added directly after line 21 to keep the article short

Reply:
I thought it was better to let people go to the Add-on part of the manual.  This is the very first thing people are going to read, and I was trying not to go into too much detail, overall.  In fact, generally, you will see that I have stripped down the article.

If you think it needs to be put back in (the three sources were there as a bulleted list before), let me know.

Reply:
I agree in general. You've both missed to consider the visually impaired. Either add a short sentence or add a title to the image (that would be a good compromise).

```
![alt text](images/distro.png "Logo Title Text 1")
```

Reply:
How did we miss?  I think you are saying that, if someone is using a text-to-speech reader they need a title for the image??  Not sure what you are getting at here.

Reply:
Sorry for being overly short in my comments once again.
Yes I am talking about screen reader users :) You have stripped the information regarding add-ons because people can read about that in the image. Is that correct? If so you could add a few words shortly explaining the image into the alt or title property of the image.

Reply:
>If you think it needs to be put back in (the three sources were there as a bulleted list before), let me know.

No you missunderstood me here.
I don't want the 3 sources back. Just one "oneliner" sentence and as @ThomDietrich already mentioned an improved alt-tag for the image, which explains what is shown to users with a screen reader.

Reply:
Just checking - so on my local repo I now have the following sentence:
"Add-ons may come from the openHAB 2 distribution, the Eclipse SmartHome project, or from the openHAB 1 distribution."  WDYT?

Should I add the following?
`![distribution overview](images/distro.png "openHAB 2 Overall Architectural View")`